.github/scripts/alpine_build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+# Copyright (C) 2025 Intel Corporation
+# Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# alpine_build.sh - Script for building UMF on Alpine image
+
+set -e
+
+cd unified-memory-framework
```

Comment:
I somehow dislike this random `cd`... but I'm not sure how to better handle this... At the moment I can only think of updating `HOST_WORKDIR` and `WORKDIR` to already include it... but not sure if this is better...?

Reply:
One or the other method is equally bizarre, so I did not make changes.